spec/v1/envelope.md

```diff
@@ -19,5 +21,41 @@ standard DSSE fields.
     specifying its schema.
 -   `payload` MUST be a base64-encoded JSON [Statement].
 
+## File naming convention
+
+Envelopes SHOULD use the suffix `.json`.
+
+-   An Envelope containing an attestation about a particular SW supply chain
+    step `<step-name>` SHOULD be named `<step-name>.json`.
+-   If multiple Envelopes are produced for the same step by different
+    [functionaries] uniquely identified by a public key, an Envelope name
+    SHOULD include the hash of the public key `<pubkey-hash>` of the signing
+    functionary: `<step-name>.<pubkey-hash>.json`.
+
+## Storage convention
+
+The media type `application/vnd.in-toto.<predicate>+<sig>` SHOULD
+be used to denote an individual attestation in arbitrary storage systems.
+
+-   The `<predicate>` MUST match the [predicate specification name]
+    without the file extension. Predicate versioning is handled in the
+    [Statement] layer.
+-   The `<sig>` MUST be a succint alias that unambiguously identifies
+    the Envelope signature format.
+-   Consumers SHOULD NOT rely upon the media type for individual attestations
+    as faithful indicators of predicate type because this information is only
+    authenticated at the [Statement] layer.
+-   To obtain predicate information that is authenticated, consumers MUST
+    parse the Envelope's `payload`.
+
+### Example
```

Comment:
optional: can we provide multiple examples?  E.g. what would the SLSA provenance or VSA type be?